gamestonk_terminal/stocks/discovery/disc_controller.py

```diff
@@ -81,6 +83,8 @@ def print_help():
     gainers        show latest top gainers
     losers         show latest top losers
     under          undervalued growth stocks
+    gtech          highest growth technology stocks
```

Comment:
Can we have:
```
gtech       technology stocks with revenue and earnings growth in excess of 25%.
active       most active stocks by intraday trade volume.
```

Would be good to have the description be a bit more accurate as explained in here: https://finance.yahoo.com/screener.